docs/general/deployment.md

```diff
@@ -119,3 +119,17 @@ _Step 4:_ commit your changes and deploy via EB CLI:
 ```sh
 eb deploy {target environment name}
 ```
+
+## Azure
+
+### Easy 3-Step Deployment Process
+_Step 1:_ Within Azure Portal, add a 'Web App' resource to your resource group.  The runtime stack must be one of the Node versions.  Please note that much of the Microsoft quick starts out result in Node being run via IIS which will not work with react-boilerplate.
```

Comment:
I'm slightly confused here. Is there an additional step necessary for making sure Node isn't being run via IIS?

Reply:
Thanks @julienben - I have added a little extra description to the step to hopefully make it a little more clear.  Let me know if it makes more sense now.